client/src/components/input.tsx

```diff
@@ -97,10 +97,10 @@ function Input(): JSX.Element {
   }
 
   // Input field keyboard was lowered
-  const onBlur = () => {
-    window.scrollTo(0, 0);
-    document.body.scrollTop = 0;
-  };
+  // const onBlur = () => {
+  //   window.scrollTo(0, 0);
+  //   document.body.scrollTop = 0;
+  // };
```

Comment:
It was scrolling the window down and not letting cypress click the send button. (double-click issue)